### ISSUE TITLE
See on github / Development repository

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#4422]
We should have a link in the tool menu that leads to the development repository, like we have for the toolshed.

**Comment by martenson** [comment]
I think `tool_shed_repository` table does not have this information so we need to enhance it, we could alternatively use the TS to do a redirect for though.

**Comment by mvdbeek** [comment]
Right, or we add it directly to the tools ... but all these options don't seem ideal, it's either more copy-'n-paste, or more reliance on the toolshed.
